drivers/SmartThings/matter-lock/src/test/test_matter_lock_batteryLevel.lua

```diff
@@ -0,0 +1,89 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local test = require "integration_test"
+local capabilities = require "st.capabilities"
+test.add_package_capability("lockAlarm.yml")
+local t_utils = require "integration_test.utils"
+local clusters = require "st.matter.clusters"
+
+local mock_device_record = {
+  profile = t_utils.get_profile_definition("lock-nocodes-notamper-batteryLevel.yml"),
+  manufacturer_info = {vendor_id = 0x101D, product_id = 0x1},
+  endpoints = {
+    {
+      endpoint_id = 2,
+      clusters = {
+        {cluster_id = clusters.Basic.ID, cluster_type = "SERVER"},
+      },
+      device_types = {
+        device_type_id = 0x0016, device_type_revision = 1, -- RootNode
+      }
+    },
+    {
+      endpoint_id = 10,
+      clusters = {
+        {cluster_id = clusters.DoorLock.ID, cluster_type = "SERVER", feature_map = 0x0000},
+        {cluster_id = clusters.PowerSource.ID, cluster_type = "SERVER"},
```

Comment:
Doesn't the PowerSource cluster here need a feature_map for the battery level? It might be good to add that since that is specifically what we are testing

Reply:
good catch